src/Models/BaseElement.php

```diff
@@ -443,9 +443,9 @@ public function getRenderTemplates()
                 break;
             }
 
-            $templates[] = $value;
-            $templates[] = 'elements/' . DataObjectPreviewController::stripNamespacing($value);
-            $templates[] = DataObjectPreviewController::stripNamespacing($value);
+            $templates[] = $value . $suffix;
+            $templates[] = 'elements/' . DataObjectPreviewController::stripNamespacing($value) . $suffix;
```

Comment:
We don't need the elements folder convention any longer since everything is namespaced. We could possibly remove this functionality altogether?

Reply:
👍